status-im/status-legacy
# Allow group creation without contacts

fixes #11139

### Summary

Now group can be created without inviting any contacts since we already have invitations via link.

#### Platforms


- Android
- iOS

##### Functional
- group chats

### Steps to test

- Open Status
- Select `start group chat`
- Ensure that `Next` button enabled when no contacts selected
- Press `Next`
- Fill in group name
- Ensure that `Create group chat` button enabled



status: ready
